box-content-sdk/src/main/java/com/box/androidsdk/content/requests/BoxRequestMetadataAdd.java

```diff
@@ -0,0 +1,37 @@
+package com.box.androidsdk.content.requests;
+
+import com.box.androidsdk.content.models.BoxMetadata;
+import com.box.androidsdk.content.models.BoxSession;
+
+import java.util.Map;
+
+/**
+ * Abstract class that represents a request to add metadata to an item on Box.
+ * @param <E>   type of BoxMetadata object returned in the response.
+ * @param <R>   type of BoxRequest to return.
+ */
+public class BoxRequestMetadataAdd<E extends BoxMetadata, R extends BoxRequest<E,R>> extends BoxRequestMetadata<E, R> {
```

Comment:
I think we can group all the metadata requests similarly to how we grouped the other requests (eg. BoxRequestsFile, BoxRequestsFolder, etc.). So I'm thinking we can move all of the individual request objects BoxRequestMetadataAdd, BoxRequestMetadataUpdate, BoxRequestMetadataDelete as static classes inside of a BoxRequestsMetadata class. So then we can instantiate them by going new BoxRequestsMetadata.Add/Update/Delete